app/routes/labs.missions/route.tsx

```diff
@@ -75,6 +76,29 @@ export default function () {
             2019 Astrophysics Senior Review, Executive Summary, Page 2
           </figcaption>
         </figure>
+        <div className="clearfix" />
+        <h3>Multimessenger Mission Timeline</h3>
+        <p className="usa-paragraph">
+          NASA missions played a critical role in the discovery and
+          characterization of the first binary neutron star merger (GW170817).
+          In the near future, the balanced mission portfolio is well-positioned
+          to continue to make major contributions to EM followup of
+          gravitational-wave sources. Current workhorse facilities in the area
+          of time-domain and multi-messenger astronony such as Fermi, Swift,
+          Chandra, & HST are well past design lifetimes. These capabiltiies are
```

Comment:
```suggestion
          Chandra, & HST are well past design lifetimes. These capabilities are
```

Also, are ampersands allowed in text in JSX, or does this need to be written as `&amp;`?

Reply:
it works fine as &
<img width="315" alt="Screenshot 2024-01-05 at 2 37 08 PM" src="https://github.com/nasa-gcn/gcn.nasa.gov/assets/6545874/596e7da7-2722-4178-ad1f-7a84e3bf6753">

Reply:
I agree. And I see that the SSR code that is being sent to the browser contains `&amp;`.